app/uk/gov/hmrc/fhregistrationfrontend/actions/UserAction.scala

```diff
@@ -99,7 +98,7 @@ case class UserAction @Inject()(
   def handleFailure(e: Throwable): Result =
     e match {
       case x: NoActiveSession ⇒
-        Logger.warn(s"could not authenticate user due to: No Active Session " + x)
+        logger.warn(s"could not authenticate user due to: No Active Session " + x)
```

Comment:
no need to `+` with string interpolation

Reply:
didn't change any code base as part of this library change. Will try to fix it